compiler/rustc_llvm/llvm-wrapper/PassWrapper.cpp

```diff
@@ -1437,9 +1437,14 @@ LLVMRustCreateThinLTOData(LLVMRustThinLTOModule *modules,
     Ret->ResolvedODR[ModuleIdentifier][GUID] = NewLinkage;
   };
 
+#if LLVM_VERSION_GE(13,0)
+  lto::Config conf;
```

Comment:
Can this have a comment describing why the default constructor is sufficient here (uses `FromPrevailing` visibility scheme which works for many binary formats).

We probably could use ELF visibility scheme for many of our targets, however.

Reply:
I have to confess I don't know enough about what's going on here to really understand: the compile-fix was  cargo-culted from https://reviews.llvm.org/D92900, and I'm happy to cargo-cult the comment from there if it applies, or drop some other comment you'd prefer, but I lack the expertise (for now) to write a good comment explaining things.

Reply:
Yeah feel free to just take something from my parens:

> Uses FromPrevailing visibility scheme which works for many binary formats. We probably could and should use ELF visibility scheme for many of our targets, however.

Something like that, anyway.